x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -372,6 +372,16 @@ export function mlApiServicesProvider(httpService: HttpService) {
       });
     },
 
+    checkIndexExists({ index }: { index: string }) {
+      const body = JSON.stringify({ index });
+
+      return httpService.http<any>({
```

Comment:
The response can be added here `{ indexExists: boolean }` rather than `any`
The response could probably just be `exists: boolean`  as the enpoint name let's you know what your getting.

Reply:
Good call - updated in e95d7ddcdae1485a589cc3b90e0bbd163ad22621